junit5/src/main/java/sunstone/core/AwsUtils.java

```diff
@@ -29,29 +28,36 @@ private static AwsCredentialsProvider getCredentialsProvider() {
 
     static boolean propertiesForAwsClientArePresent() {
         return objectProperties.getProperty(JUnit5Config.Aws.ACCESS_KEY_ID) != null
-                && objectProperties.getProperty(JUnit5Config.Aws.SECRET_ACCESS_KEY) != null
-                && objectProperties.getProperty(JUnit5Config.Aws.REGION) != null;
+                && objectProperties.getProperty(JUnit5Config.Aws.SECRET_ACCESS_KEY) != null;
     }
 
-    static CloudFormationClient getCloudFormationClient() {
+    static Region getAndCheckRegion(String regionStr) {
+        Region region = Region.of(regionStr);
+        if (region == null) {
+            throw new IllegalArgumentException("Unkown region " + regionStr);
+        }
+        return region;
+    }
+
+    static CloudFormationClient getCloudFormationClient(String region) {
```

Comment:
How expensive is creation of those clients? Often you can see in testsuites to cache clients (e.g. have just Singelton) because it use to be expensive operation. 

Is it worth in this case as well?

Reply:
Sunstone caches the clients and use only one instance. I am not sure what is the exact cost, however it opens/closes IO resources.

Testsuite (CloudTS) can create and cache its own client, however I would recommend to inject (next PR ;) ) it as a static resource - created once per test class.